tests/test_fs.py

```diff
@@ -122,3 +130,31 @@ def test_fsspec():
         data = fp.read()
     assert data == b"123", "unexpected data"
     fs.rm(filepath)
+
+
+def test_empty_directory():
+    fs = fsspec.filesystem("v3io")
+    path = f"/{test_container}/{test_dir}/test_empty_directory"
+    makedir(path)
+    try:
+        result = fs.ls(path)
+        assert result == []
+    finally:
+        fs.rmdir(path)
```

Comment:
I think we should delete test_dir after running the session - we got no benefit from remaining it None.
In order to do so without raise an error in other tests we run - we should create fixture test_dir and define scope="session" for this fixture.
Because it may be considered as extra feature - that it is not limited to this PR - I will open it as another issue.

Reply:
What does "remaining it None" mean? Sure, go ahead opening a PR or issue for it.